interview-questions.md

```diff
@@ -2,61 +2,65 @@
 
 Answer the following questions. First, without external resources. Challenge yourself to answer from memory. Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn.
 
-1. Why would you use the method super()?
+1. Why would you use the method super()
 
-  Your answer:
-
-  Researched answer:
+  Your answer:you would use the method super to import all the keyvalue pairs and the starter javascrpit from a parent class to a child class
 
+  Researched answer:super() will  the constructor of its parent class. 
 
 
 2. What is a virtual DOM?
 
-  Your answer:
-
-  Researched answer:
+  Your answer:  The virtual DOM is a ectracted from the Dom. it is used in React
 
+  Researched answer:Perhaps it’s better to think of the virtual DOM as React’s local and simplified copy of the HTML DOM. It allows React to do its computations within this abstract world and skip the “real” DOM operations, often slow and browser-specific.
 
 
 3. When creating a basic (stateless) class component in React, what are the necessary elements needed to display "Hello World" in the browser?
 
-  Your answer:
+  Your answer:you need a function and a return
 
-  Researched answer:
+  Researched answer:A stateless component is just a plain javascript function which takes props as an argument and returns a react element.
```

Comment:
yes! this case would be a functional component. you can also have a stateless class component if you'd like